### DISCUSSION TITLE
OMI glTF Working Group Meeting 2022-09-29

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/gltf-extensions#108] [General]
This meeting will be on 9/29/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Last Week's Agenda for reference:

- glTF Transform Updates
- [OMI_collider](https://github.com/omigroup/gltf-extensions/pull/63)
- [KHR_audio / Khronos feedback](https://github.com/KhronosGroup/glTF/pull/2137)
- [glXF](https://github.com/KhronosGroup/glXF/blob/main/specification/2.0/README.md)
  - OMI_id / [OMI_ref](https://github.com/omigroup/gltf-extensions/pull/85)
- USDz spike
- Schedule alternate timezone meeting
- Show and tell
- Create next weeks agenda

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by robertlong** [comment]
# Notes:

- KHR_audio progressing in Khronos ratification track. Needs sample models to continue
- glXF discussion
  - @robertlong pushing for glXF to include the ability to compose materials/animations etc.
  - Where does USD fit into this space? 
  - https://github.com/tsturm/glTF/tree/FRAUNHOFER_external_model/extensions/2.0/Vendor/FRAUNHOFER_external_model
  - Comparisons to Webaverse's metaversefiles.
- Talk about federated glTF asset repositories
  - https://api.icosa.gallery/docs
  - https://spec.matrix.org/v1.4/client-server-api/#get_matrixmediav3downloadservernamemediaid
  - glTF 3D Commerce also thinking about related things
  - MSF as well: https://docs.google.com/document/d/1w-hnAtizCBj2foBdLj60ZJzVSmToc5iDFGknrONiSlc/edit#
